sys/shell/commands/sc_netif.c

```diff
@@ -218,14 +218,14 @@ static void _netif_set_u16(kernel_pid_t dev, ng_netconf_opt_t opt,
         if ((res = strtoul(u16_str, NULL, 10)) == ULONG_MAX) {
             puts("error: unable to parse value.\n"
                  "Must be a 16-bit unsigned integer (dec or hex)\n");
-            return;
+            return 1;
```

Comment:
I thought at least some values are specified by POSIX (but maybe it's just the name). Also, the shell doesn't necessarily use a plaintext protocol for I/O. However, I don't want to advocate for the use of errno values. Just wanted to point out that it has been used in RIOT context - though not in RIOT itself - for this purpose.